frontend/src/components/Gallery.tsx

```diff
@@ -1,38 +1,36 @@
 import Card from "./Card";
 import "./Gallery.scss"
 import {useEffect, useState} from "react";
-import {fetch10Characters, fetch1Characters} from "../services/RequestService";
+
+import {fetch1Characters} from "../services/RequestService";
 
 export default function Gallery(){
-    const [Animals10, setAnimals10] = useState<any>([]);
     const [Animals1, setAnimals1] = useState<any>([]);
```

Comment:
What does the 1 stand for?

```suggestion
    const [animals1, setAnimals1] = useState<any>([]);
```

Reply:
already renamed, sorry